packages/react/src/components/searchInput/SearchInput.stories.tsx

```diff
@@ -214,3 +214,15 @@ WithSuggestionsSpinner.args = {
   label: 'Search for a fruit',
   helperText: 'Assistive text',
 };
+
+export const WithPlaceholder = (args) => {
```

Comment:
Instead of adding a new story to demo this, I think we should add the placeholder text to other stories. E.g. see how this is demoed for the TextInput for example.

Reply:
Good idea! 👍